FinancialTransactionsApi/V1/Gateways/DynamoDbGateway.cs

```diff
@@ -38,78 +40,47 @@ public async Task AddRangeAsync(List<Transaction> transactions)
             }
         }
 
-        public async Task<TransactionList> GetAllTransactionsAsync(TransactionQuery query)
+        public async Task<TransactionList> GetPagedTransactionsAsync(TransactionQuery query)
```

Comment:
Hi Edward, can you please have a look at this example https://github.com/LBHackney-IT/notes-api/blob/master/NotesApi/V2/Gateways/NotesDbGateway.cs for implementing pagination so we can keep it consistent. Also, this method could be consolidated with GetTransactionsAsync.

Reply:
Done